tests/conftest.py

```diff
@@ -1487,11 +1493,17 @@ def finalizer():
                     else:
                         ceph_health_check_base()
                         log.info("Ceph health check passed at teardown")
+                        log.info("Removing Faster recovery profile")
+                        ceph_cmd = "ceph config set osd osd_mclock_profile balanced"
```

Comment:
You can rename it to `ceph_remove_cmd`, move above line 1470 and remove the duplicated line from 1505.

Reply:
sure will do